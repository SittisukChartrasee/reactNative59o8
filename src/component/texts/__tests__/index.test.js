import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components/native'
import {
  TThin,
  TText,
  TMed,
  TLight,
  TBold,
  TSemiBold,
} from '../index'

it('renders TThin', () => {
  const tree = renderer.create(<TThin />).toJSON()
  expect(tree).toMatchSnapshot()
  renderer.create(<TThin />)
})
it('renders TText', () => {
  const tree = renderer.create(<TText />).toJSON()
  expect(tree).toMatchSnapshot()
  renderer.create(<TText />)
})
it('renders TMed', () => {
  const tree = renderer.create(<TMed />).toJSON()
  expect(tree).toMatchSnapshot()
  renderer.create(<TMed />)
})
it('renders TLight', () => {
  const tree = renderer.create(<TLight />).toJSON()
  expect(tree).toMatchSnapshot()
  renderer.create(<TLight />)
})
it('renders TBold', () => {
  const tree = renderer.create(<TBold />).toJSON()
  expect(tree).toMatchSnapshot()
  renderer.create(<TBold />)
})
it('renders TSemiBold', () => {
  const tree = renderer.create(<TSemiBold />).toJSON()
  expect(tree).toMatchSnapshot()
  renderer.create(<TSemiBold />)
})