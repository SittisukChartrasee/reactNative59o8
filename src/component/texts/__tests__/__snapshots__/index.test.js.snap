// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TBold 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-Bold",
        "fontSize": 18,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`renders TLight 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-Light",
        "fontSize": 16,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`renders TMed 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-Medium",
        "fontSize": 18,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`renders TSemiBold 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-SemiBold",
        "fontSize": 18,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`renders TText 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-Text",
        "fontSize": 18,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`renders TThin 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-Thin",
        "fontSize": 18,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "SukhumvitSet-Thin",
        "fontSize": 18,
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "textAlign": "center",
      },
    ]
  }
/>
`;
